<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaJugo</class>
 <widget class="QMainWindow" name="ventanaJugo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(255, 220, 150, 255), /* Color claro, hacia la izquierda */
        stop:1 rgba(230, 134, 60, 255)  /* Color naranja oscuro, hacia la derecha */
    );

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="jPinaCooler">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/pinaCooler.png);
background: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jNaranjaSunrise">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/naranjaSunrise.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jTitulo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>900</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-family: &quot;Calibri&quot;;         /* Fuente Calibri */
    font-size: 18pt;                /* Tamaño de 20pt */
    color: #AC1F01;                 /* Color rojo oscuro especificado */
    font-weight: bold;            /* Normal, ajusta a bold si necesitas */
    text-align: left;               /* Alineación izquierda, cámbialo a center si es necesario */
	background: none;

}
</string>
    </property>
    <property name="text">
     <string>Opciones de Vodka disponibles… Seleccione su favorito</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jLimonada">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/limonada.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jNaranjada">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/naranjada.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jCitrusPunch">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/citrusPunch.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jLimonadaEspumosa">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/limonadaEspumosa.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jPinaColada">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>720</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasJugo/pinaColada.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="jBoton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>900</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e67e22;
    color: white;                /* Texto blanco */
    font-family: Calibri;         /* Fuente Calibri */
    font-size: 22px;              /* Tamaño de la fuente */
    font-weight: bold;  
    border-radius: 20px;          /* Bordes redondeados */
    padding: 10px 20px;           /* Espaciado interno para agrandar el botón */
}

QPushButton:hover {
   background-color: #d35400;     /* Color de fondo al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Bebidas/bebidasJugo/bebidasJugo.qrc"/>
 </resources>
 <connections/>
</ui>
